commands/filescommands/objectcommands/uploaddir.go

```diff
@@ -0,0 +1,235 @@
+package objectcommands
```

Comment:
we had a brief chat about this in slack. it seemed better suited to `objectcommands` because I think the vast majority of use cases will be to upload objects, rather than empty pseudo-directories. what do you think @kenperkins?